Assets/PatchKit Patcher/Scripts/UI/Languages/TextTranslator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    [RequireComponent (typeof(Text))]
+    public class TextTranslator : MonoBehaviour
+    {
+        public string Key;
+        private Text _text;
+
+        void Awake ()
+        {
+            _text = GetComponent<Text>();
+        }
+        
+        void Start ()
+        {
+            _text.text = PatcherLanguages.GetTraduction (Key);
```

Comment:
Unnecessary space before the parenthesis.

Reply:
done